Copilot backlog handling: Fix `authorId` → `author` route parameter name mismatch in custom-model-binding article (#36793)
dotnet/AspNetCore.Docs: aspnetcore/mvc/advanced/custom-model-binding.md

Reviewer: ```suggestion
You can POST a base64-encoded string to the previous API method using a tool like [curl](https://curl.haxx.se/).
```

Reviewer: ```suggestion
> [!NOTE]
```

[!code-csharp[](custom-model-binding/samples/3.x/CustomModelBindingSample/Controllers/BoundAuthorsController.cs?name=snippet_GetById&highlight=2)]

> In this example, since the name of the argument isn't the default `author`, it's specified on the parameter using the `ModelBinder` attribute. Both the controller and action method are simplified compared to looking up the entity in the action method. The logic to fetch the author using Entity Framework Core is moved to the model binder. This can be a considerable simplification when you have several methods that bind to the `Author` model.

Reviewer: ```suggestion
In this example, since the route parameter name (`id`) doesn't match the action method parameter name (`author`), the `Name` property on the `ModelBinder` attribute is used to specify which route value to bind. Both the controller and action method are simplified compared to looking up the entity in the action method. The logic to fetch the author using Entity Framework Core is moved to the model binder. This can be a considerable simplification when you have several methods that bind to the `Author` model.
```
